coalton.asd

```diff
@@ -268,11 +268,11 @@
                (:file "fibonacci")
                (:file "big-float")
                (:module "gabriel-benchmarks"
-                        :serial t
```

Comment:
My auto-formatter hit this, seemed like a mistake

Reply:
your formatting is the correct one